### PR TITLE
Log socket error string when socket error happens

### PR DESCRIPTION
**Pull request by roxanneskelly**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19769/HighFidelity-Beta-PR15677-512b43e.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19769/HighFidelity-Quest-Beta-PR15677-512b43e.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19769/HighFidelity-Beta-PR15677-512b43e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19769/HighFidelity-Beta-Interface-PR15677-512b43e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19769/HighFidelity-Beta-PR15677-512b43e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19769/HighFidelity-Beta-Interface-PR15677-512b43e.exe

File: libraries/networking/src/udt/Socket.cpp [hunk 1]
```diff
@@ -486,7 +486,11 @@ std::vector<HifiSockAddr> Socket::getConnectionSockAddrs() {
 }
 
 void Socket::handleSocketError(QAbstractSocket::SocketError socketError) {
-    qCDebug(networking) << "udt::Socket error - " << socketError;
+    qCDebug(networking) << "udt::Socket (" << _udpSocket.state() << ") error - " << socketError << "(" << _udpSocket.errorString() << ")";
+#ifdef DEBUG_EVENT_QUEUE
```
Comment (howard-stearns) [inline libraries/networking/src/udt/Socket.cpp]: Interesting idea re event queue. 
handleSocketError is connected to an error signal, while the error case of messaging in writeDatagram is sychornous.
So I think this one MIGHT grow as errors accumulate if they come fast enough, but the very first error in writeDatagram might be the only place to know if we're being delayed for some reason. But I'm not sure how/what that would be.

Thoughts?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19780/HighFidelity-Beta-PR15677-04f879b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19780/HighFidelity-Beta-Interface-PR15677-04f879b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19780/HighFidelity-Beta-PR15677-04f879b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19780/HighFidelity-Beta-Interface-PR15677-04f879b.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19780/HighFidelity-Beta-PR15677-04f879b.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19780/HighFidelity-Quest-Beta-PR15677-04f879b.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19785/HighFidelity-Beta-PR15677-2f2e967.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19785/HighFidelity-Beta-Interface-PR15677-2f2e967.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19785/HighFidelity-Beta-PR15677-2f2e967.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19785/HighFidelity-Quest-Beta-PR15677-2f2e967.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19785/HighFidelity-Beta-PR15677-2f2e967.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19785/HighFidelity-Beta-Interface-PR15677-2f2e967.exe

**Pull request merged**